indexer/ftypes_matcher.cpp

```diff
@@ -461,8 +461,11 @@ TwoLevelPOIChecker::TwoLevelPOIChecker() : ftypes::BaseChecker(2 /* level */)
       {"man_made", "water_tap"},
       {"man_made", "water_well"},
       {"natural", "beach"},
-      {"natural", "geyser"},
       {"natural", "cave_entrance"},
+      {"natural", "geyser"},
+      {"natural", "hot_spring"},
+      {"natural", "peak"},
+      {"natural", "saddle"},
```

Comment:
I'd prefer not to treat peak, saddle as POIs.

Reply:
What does it affect besides the search?

Reply:
IsPoiChecker

Reply:
Yes, but what is it used for? :)

Reply:
Search ranking, Editor, Wiki builder, some lists, 6 occurencies.

Reply:
I've checked all usages I could find (see my new comments) and I see no issue with adding peaks/saddles.